synapse/http/client.py

```diff
@@ -371,7 +371,7 @@ def post_json_get_json(self, uri, post_json, headers=None):
         Args:
             uri (str):
             post_json (object):
-            headers (dict[str, List[str]]|None): If not None, a map from
+            headers (dict[bytes, List[str]]|None): If not None, a map from
```

Comment:
I think this has got lost
```suggestion
            headers (dict[str, List[str]]|None): If not None, a map from
```